### PR TITLE
fix(image): introduce a new setting `start-pre-pull-manager-images`

### PR DESCRIPTION
**Pull request by mantissahz**
#### Which issue(s) this PR fixes:

Issue # longhorn/longhorn#8376

### CODE REVIEW

File: content/docs/1.7.0/references/helm-values.md [hunk 1]
```diff
@@ -250,6 +250,7 @@ During installation, you can either allow Longhorn to use the default system set
 | defaultSettings.snapshotDataIntegrityCronjob | Setting that defines when Longhorn checks the integrity of data in snapshot disk files. You must use the Unix cron expression format. |
 | defaultSettings.snapshotDataIntegrityImmediateCheckAfterSnapshotCreation | Setting that allows disabling of snapshot hashing after snapshot creation to minimize impact on system performance. |
 | defaultSettings.snapshotMaxCount | Maximum snapshot count for a volume. The value should be between 2 to 250 |
+| defaultSettings.startPrePullManagerImages | Pre-pull Longhorn manager images on all worker nodes before starting the manager pods. |
```
Comment (jillian-maroket) [inline content/docs/1.7.0/references/helm-values.md:253]:
```suggestion
| defaultSettings.startPrePullManagerImages | Setting that allows worker nodes to pull Longhorn Manager images before Longhorn Manager pods are started. |
```

File: content/docs/1.7.0/references/settings.md [hunk 2]
```diff
@@ -456,6 +457,12 @@ The setting enables fast replica rebuilding feature. It relies on the checksums
 
 The value in seconds specifies the timeout of the HTTP client to the replica's file sync server used for replica rebuilding, volume cloning, snapshot cloning, etc.
 
+#### Pre-Pull Manager Images
+
+> Default: `true`
+
+The setting enables pre-pulling Longhorn managers' images on all worker nodes before starting the managers' pods.
```
Comment (jillian-maroket) [inline content/docs/1.7.0/references/settings.md:464]:
```suggestion
Setting that allows worker nodes to pull Longhorn Manager images before Longhorn Manager pods are started.
```

**Review by jillian-maroket — COMMENTED** [review]
Let me know if you have concerns about the changes.

**Comment by mantissahz** [comment]
we do not need this setting now.

**Pull request closed without merging**